traffic_portal/test/integration/PageObjects/CacheGroup.po.ts

```diff
@@ -158,6 +156,26 @@ export class CacheGroupPage extends BasePage {
         await snp.NavigateToCacheGroupsPage();
         return result;
     }
+    public async CheckCSV(name: string): Promise<boolean> {
+        return element(by.cssContainingText("span", name)).isPresent();
+    }
+    public async ToggleTableColumn(name: string): Promise<boolean> {
+        let result = false;
+        await this.btnTableColumn.click();
+        //if it's already on, turn it off and return false
+        if (await browser.isElementPresent(element(by.xpath("//th[text()='" + name + "']"))) === true){
```

Comment:
XPath is unfamiliar and confusing - especially because HTML is not an XML dialect, so for some valid HTML XPath queries would be fundamentally incapable of selecting certain tags. Those situations are a bit contrived, but in general web developers are much, much more likely to be familiar with CSS selectors than XPath.

This `by` directive can be written more simply and shorter as just `by.cssContainingText("th", name)`.